modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-updating-cluster-manifests_{context}"]
+= Installing the ImageContentSourcePolicy and CatalogSource resources into the cluster
+
+// TODO: Can we find a better title / more generic than calling out the two resources?
+
+After you have mirrored your image set to the mirror registry, you must install the generated `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
+
+// TODO: why?
```

Comment:
@afflom @jpower432 Can you explain in simple terms what exactly this is doing / why this is needed? The google doc says "oc-mirror will output kubernetes manifests that are required by OpenShift for nodes to utilize the images within the mirror registry. " but it's still not clear to me.

Reply:
Sure! Since `oc-mirror` is tracking which images are being requested originally and where they are published in the target mirror registry, it creates `ImageContentSourcePolicy` (ICSP) custom resources to define the mirror location for the original images. This is especially helpful when it comes to utilizing the custom catalog images being built. `oc-mirror` generates a `CatalogSource` to allow the catalog to be utilized by the cluster; the operators images being referenced by the catalogs are still defined as their original image location. The ICSP allows the source and mirrored location to be checked for the operator container image. Does this help? Perhaps it might be useful to link to ICSP documentation. WDYT?

Reply:
Thanks @jpower432 that helps. I took a look at elsewhere that we talk about ICSPs and catalog sources and took a stab at a description for them. Can you take a look at what I say now and correct anything that's not right?

```
 The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects container pull requests from the online registries to the mirror registry. The `CatalogSource` resource is used by OLM to retrieve information about the available Operators in the mirror registry.
```

Preview: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-updating-cluster-manifests_installing-mirroring-disconnected

Reply:
@bergerhoffer One nit here:  I would replace `container` with `image`.